### ISSUE TITLE
Directory structure

### ISSUE THREAD

**Issue by triazo** [triazo/EvoOS#7]
To keep things seperate we can have a directory for each module. I doubt we'll use everything specified below for a bit but I wanted to get it out there. Each module should be simple enough for now to fit in one directory.
- EvoOS
  - Dispatcher
  - Modules
    - WebClient
    - Julius
  - LibraryA
  - LibraryB

**Comment by not-inept** [comment]
++

**Comment by not-inept** [comment]
We should reorganize main to reflect this sooner rather than later, also I'd prefer a lib (or libraries) folder to contain folders per library directories, and lowercase directory names. I'll call out on IRC for consensus. Should we do a 3/4 people must respond to close an issue, or 4/4?

**Comment by triazo** [comment]
nah, just close the issue when you think its done. Lets not formalize too much about the github stuff. This one I'll probably leave open indefinitely for the discussion, in case it ever needs to change. My view on issues is more that they're a discussion forum than a tracker. If an issue is worth being tracked it is worth discussing, otherwise just fix it.

Lets also try to keep the amount of libraries we need down. For all of our current tasks we won't need libraries. Putting them in the top level directory should force us to think twice about what libraries are really necessary.

**Comment by droghio** [comment]
I'll take charge of the reorg, I'm gonna have to patch up what we currently have anyway so it isn't much extra.

**Comment by triazo** [comment]
This is still open for discussion and always will be, even though action is currently being taken. The fact that things are going to be done on an issue should be implicit, so I don't see the need for a todo tag.

**Comment by not-inept** [comment]
I was using the todo tag to mark things when there is an expected outcome of some sort that has been promised and we are waiting for. Currently we're implementing the results of the discussion, though I do agree this will be an active process.

I'm still inclined as I was on IRC, towards separating modules to their own repos and using githubs file structure for that, I've added to the wiki to keep it noted somewhere as we continue working on other elements.

**Comment by triazo** [comment]
fwiw github's structure is username/repositiory

Doing it that way would require making seperate organizations. I don't want to overorganize.
